content/_code-samples/send-a-memo/py/send-a-memo.py

```diff
@@ -0,0 +1,61 @@
+from xrpl.clients import JsonRpcClient
+from xrpl.models.transactions import Payment, Memo
+from xrpl.transaction import safe_sign_and_autofill_transaction, send_reliable_submission
+from xrpl.wallet import generate_faucet_wallet
+
+# This code sample validates and sends a transaction with a Memo attached
+# A funded account on the testnet is provided for testing purposes
+# https://xrpl.org/transaction-common-fields.html#memos-field
+
+# Connect to a testnet node
+JSON_RPC_URL = "https://s.altnet.rippletest.net:51234/"
+client = JsonRpcClient(JSON_RPC_URL)
+
+# Get account credentials from the Testnet Faucet
+print("Requesting account from the Testnet faucet...")
+test_wallet = generate_faucet_wallet(client=client)
+myAddr = test_wallet.classic_address
+
+# Example: TEXT 321 -+=
+memo_data = input("Enter memo data to insert into a transaction: ")
+# Example: text
+memo_type = input("Enter memo type to insert into a transaction: ")
+
+memo_data = memo_data.encode('utf-8').hex()
+memo_type = memo_type.encode('utf-8').hex()
+memo_format = "text/csv".encode('utf-8').hex()
```

Comment:
Might be worth linking to where these `MemoFormat` values can be found. Applies to both files. (I think this is one site you could link? - https://developer.mozilla.org/en-US/docs/Web/HTTP/Basics_of_HTTP/MIME_types/Common_types)